### PR TITLE
Oc client jadeless

### PR DESCRIPTION
**Pull request by matteofigus**
This is a short term fix for #297 

Given oc-client requires Jade only for its own client-side runtime, and this is going to be removed soon anyway in favour of Pug on #300 (which compiles encapsulated templates so runtime is not gonna be needed anymore) - this should fix some problems as insecure dependencies and npm shrinkwrap. In fact, npm-shrinkwrap is not actually needed anymore in the oc-client as it was a temporary measure to get snyk green.

### CODE REVIEW

File: client/src/executor.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var vm = require('vm');
 
 module.exports = {
   template: function(code, template){
-    var context = template.type === 'jade' ? { jade: require('jade/runtime.js')} : {};
+    var context = template.type === 'jade' ? { jade: require('./renderers/support/jade-runtime.js')} : {};
```
Comment (mattiaerre) [inline client/src/executor.js]: so it looks like we are including the `jade-runtime` lib as a static resource right?

File: client/src/renderers/support/jade-runtime.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.jade = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```
Comment (mattiaerre) [inline client/src/renderers/support/jade-runtime.js]: it this the file that you would find in `/node_modules/jade/lib/runtime.js`?

Comment (matteofigus) [inline client/src/renderers/support/jade-runtime.js]: No, this is the one you would find in `/node_modules/jade/runtime.js`

File: client/src/executor.js (hunk 1 above)
Comment (matteofigus) [inline client/src/executor.js]: True, this is the only thing I need from Jade. Including it as whole dependency means including a series of sub-dependencies I don't care about which contains some known security vulnerabilities - which is the reason we shrinkwrapped the deps in the first place. This is going to be cleaned-up as part of the Pug migration.

File: client/src/renderers/support/jade-runtime.js (hunk 2 above)
Comment (mattiaerre) [inline client/src/renderers/support/jade-runtime.js]: kk gotcha!

**Comment by mattiaerre** [comment]
LGTM 👍

**Pull request merged**